backend/src/test/java/ai/verta/modeldb/ExperimentTest.java

```diff
@@ -100,6 +101,12 @@ public static void setServerAndService() throws Exception {
         serverBuilder, databasePropMap, propertiesMap, authService, roleService);
     serverBuilder.intercept(new ModelDBAuthInterceptor());
 
+    Map<String, Object> testUerPropMap = (Map<String, Object>) testPropMap.get("testUsers");
+    if (testUerPropMap != null && testUerPropMap.size() > 0) {
+      authClientInterceptor = new AuthClientInterceptor(testPropMap);
+      channelBuilder.intercept(authClientInterceptor.getClient1AuthInterceptor());
+    }
+
```

Comment:
Broken this due to removed  https://github.com/VertaAI/modeldb/commit/b490fc6d8ddc0f3551e80a05b17b7b95c452981b#diff-c905410e9201c661229075899cc6154397fd5137aa1d2c5fd255628fc35e6041L101-L107 code by mistake.